data/sql/postgres_example.sql

```diff
@@ -1,18 +1,4 @@
 -- Example Postgres schema matching the dataset in public/data/dataset/postgres_example_dataset.yml
-DROP TABLE IF EXISTS public.report;
```

Comment:
We are dropping tables before creating them, we're just doing it a different way now.

Postgres doesn't like `DROP DATABASE ...` in the context of transactions. This doesn't play nicely with our SQLAlchemy sessions. This isn't an issue anywhere else in the codebase because well, we'd never want to execute a statement like that. In this case I've moved the drop DB logic into the fixture itself, this is predominantly to get around the issue where `TRUNCATE ...` statements were locking the DB indefinitely in the tests.